packages/@aws-cdk/aws-cloudtrail/lib/index.ts

```diff
@@ -122,30 +128,36 @@ export class Trail extends Resource {
    */
   public readonly trailSnsTopicArn: string;
 
+  private s3bucket: s3.IBucket;
   private eventSelectors: EventSelector[] = [];
 
   constructor(scope: Construct, id: string, props: TrailProps = {}) {
     super(scope, id, {
       physicalName: props.trailName,
     });
 
-    const s3bucket = new s3.Bucket(this, 'S3', {encryption: s3.BucketEncryption.UNENCRYPTED});
     const cloudTrailPrincipal = new iam.ServicePrincipal("cloudtrail.amazonaws.com");
 
-    s3bucket.addToResourcePolicy(new iam.PolicyStatement({
-      resources: [s3bucket.bucketArn],
-      actions: ['s3:GetBucketAcl'],
-      principals: [cloudTrailPrincipal],
-    }));
-
-    s3bucket.addToResourcePolicy(new iam.PolicyStatement({
-      resources: [s3bucket.arnForObjects(`AWSLogs/${Stack.of(this).account}/*`)],
-      actions: ["s3:PutObject"],
-      principals: [cloudTrailPrincipal],
-      conditions:  {
-        StringEquals: {'s3:x-amz-acl': "bucket-owner-full-control"}
-      }
-    }));
+    if (props.s3Bucket === undefined) {
```

Comment:
I think this can be made simpler by writing something like:

```ts
this.s3bucket = props.bucket || new s3.Bucket(...);
```

Same as we do for roles all over the place.

Reply:
This would then cause the policy to be applied to the bucket , the intent was to not apply the policy .

Reply:
@rix0rrr  I have applied the change as suggested.